Demo/Demo/FeatureViewControllers/PaymentButtonCustomizationViewController.swift

```diff
@@ -0,0 +1,221 @@
+import UIKit
+import PayPalUI
+
+class PaymentButtonCustomizationViewController: UIViewController {
+
+
+    // MARK: - Views
+
+    /// Scroll view which will auto size
+    let scrollView = UIScrollView(frame: .zero)
+
+    lazy var paymentButton: PaymentButton = setupPaymentButton()
+
+    lazy var spbFundingPicker: UISegmentedControl = {
```

Comment:
```suggestion
    lazy var fundingPicker: UISegmentedControl = {
```
Same here, the `spb` prefix isn't very expressive and we can align with [naming conventions](https://www.swift.org/documentation/api-design-guidelines/#avoid-abbreviations) here as well.

Reply:
btw, what's SPB?

Reply:
@kgangineni achronym for Smart Payment Button